os/lib/heapmem.h

```diff
@@ -164,6 +162,13 @@ bool heapmem_free(void *ptr);
 
 void heapmem_stats(heapmem_stats_t *stats);
 
+/**
+ * \brief       Obtain the minimum alignment of allocated addresses.
+ * \return      The alignment value that is a power of two.
+ */
+
+unsigned heapmem_alignment(void);
```

Comment:
The C11 `_Alignof` returns `size_t`, is there any reason to use a different type for heapmem alignment than what is used in the next C standard?